requirements/requirements_ansible.txt

```diff
@@ -50,7 +50,7 @@ deprecation==2.0          # via openstacksdk
 docutils==0.14            # via botocore
 dogpile.cache==0.6.5      # via openstacksdk
 entrypoints==0.2.3        # via keyring
-enum34==1.1.6             # via cryptography, knack, msrest, ovirt-engine-sdk-python
+enum34==1.1.6; python_version < '3' # via cryptography, knack, msrest, ovirt-engine-sdk-python
```

Comment:
This is a polyfill that makes `import enum` work in py2; installing it in py3 causes problems.